mne/viz/decoding.py

```diff
@@ -11,6 +11,127 @@
 import numpy as np
 import warnings
 
+def plot_topopatterns(csp, layout=None, vmin=None, vmax=None, sensors=True,
```

Comment:
I would call it plot_csp_patterns and make this feature accessible as method of CSP class